pages/1.13/security/oss/iam-api/index.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: layout.pug
+navigationTitle:  Identity and Access Management API
+title: Identity and Access Management API
+menuWeight: 110
+excerpt: Managing users with the IAM API
+
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+The Identity and Access Management API allows you to manage users through a RESTful interface. It offers more functionality than the DC/OS web interface.
+
+
+# Request and response format
+
+The API supports JSON only. You must include `application/json` as your `Content-Type` in the HTTP header, as shown below.
+
+    Content-Type: application/json
+
+
+# Host name and base path
+
+The host name to use will vary depending on where your app is running.
+
+* If your app runs outside of the DC/OS cluster, you should use the cluster URL. This can be obtained by launching the DC/OS web interface and copying the domain name from the browser. Alternatively, you can log in to the DC/OS CLI and type `dcos config show core.dcos_url` to get the cluster URL. In a production environment, this should be the path to the load balancer which sits in front of your masters.
+
+* If your app runs inside of the cluster, use `master.mesos`.
+
+Append `/acs/api/v1` to the host name, as shown below.
+
+    https://<host-ip>/acs/api/v1
+
+
+# Authentication and authorization
+
+All IAM endpoints require an authentication token---except the `auth` endpoints. The `auth` endpoints do not require authentication tokens because their purpose is to return authentication tokens upon successful login.
```

Comment:
The caption mentions "authorization". The current authorization model in open DC/OS is that all authenticated requests are authorized to do everything. It's worth explaining that (or we should remove "authorization" from the heading).